test/integration/subnet_test.rb

```diff
@@ -17,8 +22,11 @@ class SubnetIntegrationTest < ActionDispatch::IntegrationTest
   test "edit page" do
     visit subnets_path
     click_link "one"
+    assert page.has_link?('Parameters', :href => '#params')
     fill_in "subnet_name", :with => "one-secure"
     fill_in "subnet_network", :with => "10.1.1.177"
+    click_link 'Parameters'
+    assert page.has_content? 'subnet_param_value'
```

Comment:
This is failing, I suggest to remove it. 
The other parameters don't have the test either and to truly check it you need to add a parameter and then edit the same one. I think the create test is enough here.